### PR TITLE
Small changes to identify bots page

### PR DESCRIPTION
**Pull request by MargotLepizzera**
### What does this PR do?

This PR performs a few changes on the "identify bots" page for synthetic tests, notably to add a link to EU ip-ranges page.

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 1]
```diff
@@ -21,39 +21,58 @@ further_reading:
...
-Use the header attached to Datadog robots to detect them for your API and browser tests:
+You can use the [**Synthetic Monitoring IP ranges**][1] corresponding to each Datadog managed location.
+
+[1]: https://ip-ranges.datadoghq.com/synthetics.json
+
+{{< /site-region >}}
+
+{{< site-region region="eu" >}}
+
+You can use the [IP addresses][1] corresponding to each Datadog managed location.
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/identify_synthetics_bots.md:38]:
```suggestion
You can use the [**Synthetic Monitoring IP ranges**][1] corresponding to each Datadog managed location.
```

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 2]
```diff
@@ -21,39 +21,58 @@ further_reading:
...
+
+You can use the [**Synthetic Monitoring IP ranges**][1] corresponding to each Datadog managed location.
+
+[1]: https://ip-ranges.datadoghq.eu/synthetics.json
+
+{{< /site-region >}}
+
+## Default headers
+
+You can also identify Datadog robots by using some of the default headers attached to the requests generated by API and browser tests:
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/identify_synthetics_bots.md:46]:
```suggestion
You can also identify Datadog robots by using some of the **default headers** attached to the requests generated by API and browser tests:
```

**Review by amymariaparker2401 — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

**Comment by amymariaparker2401** [comment]
margot.lepizzera/identify_bots_changes

**Comment by amymariaparker2401** [comment]
[log.tar.gz](https://github.com/DataDog/documentation/files/5576734/log.tar.gz)

**Pull request merged**